Dockerfile.openpilot

```diff
@@ -13,18 +13,19 @@ COPY SConstruct ${OPENPILOT_PATH}
 COPY ./openpilot ${OPENPILOT_PATH}/openpilot
 COPY ./third_party ${OPENPILOT_PATH}/third_party
 COPY ./site_scons ${OPENPILOT_PATH}/site_scons
-COPY ./rednose ${OPENPILOT_PATH}/rednose
-COPY ./rednose_repo/site_scons ${OPENPILOT_PATH}/rednose_repo/site_scons
+COPY ./rednose_repo ${OPENPILOT_PATH}/rednose_repo
 COPY ./tools ${OPENPILOT_PATH}/tools
 COPY ./release ${OPENPILOT_PATH}/release
 COPY ./common ${OPENPILOT_PATH}/common
 COPY ./opendbc ${OPENPILOT_PATH}/opendbc
 COPY ./cereal ${OPENPILOT_PATH}/cereal
 COPY ./msgq_repo ${OPENPILOT_PATH}/msgq_repo
-COPY ./msgq ${OPENPILOT_PATH}/msgq
 COPY ./panda ${OPENPILOT_PATH}/panda
 COPY ./selfdrive ${OPENPILOT_PATH}/selfdrive
 COPY ./system ${OPENPILOT_PATH}/system
 COPY ./body ${OPENPILOT_PATH}/body
 
+RUN ln -s ${OPENPILOT_PATH}/rednose_repo/rednose ${OPENPILOT_PATH}/rednose
+RUN ln -s ${OPENPILOT_PATH}/msgq_repo/msgq ${OPENPILOT_PATH}/msgq
```

Comment:
why doesn't copying the symlinks work

Reply:
Looks like `COPY` always resolves the links

Reply:
can you just copy openpilot/ then?

Reply:
but only when copying individual files, it preserves them when copying a directory containing links

Reply:
> can you just copy openpilot/ then?

yeah, `COPY . ${OPENPILOT_PATH}/` will work, assuming that the dir is clean, which should be fine in CI